ad-tag/source/ts/types/prebidjs.ts

```diff
@@ -1995,6 +1996,36 @@ export namespace prebidjs {
       readonly excludeEvents?: string[];
     }
 
+    /**
+     * @see https://docs.prebid.org/dev-docs/modules/genericAnalyticsAdapter.html
+     */
+    export interface IGenericAnalyticsAdapterOptions {
+      /**
+       * Required. URL of the endpoint for data collection
+       */
+      url: string;
+
```

Comment:
Please add the other parameters as well

* `gvlid`
* `batchDelay`
* `sampling`
* `method`